src/ProjectTask_Ticket.php

```diff
@@ -59,6 +59,23 @@ public function getForbiddenStandardMassiveAction()
         return $forbidden;
     }
 
+    public function prepareInputForAdd($input)
+    {
+        if (
+            countElementsInTable(
+                static::getTable(),
+                [
+                    static::$items_id_1 => $input[static::$items_id_1] ?? 0,
+                    static::$items_id_2 => $input[static::$items_id_2] ?? 0,
+                ]
+            ) > 0
+        ) {
+            return false;
```

Comment:
Please add a message, to ease the debugging.

```suggestion
            Session::addMessageAfterRedirect(__s('Relation already exists.'), false, ERROR);
            return false;
```